plugins/pipelines/app/views/samson_pipelines/_fields.html.erb

```diff
@@ -0,0 +1,23 @@
+<fieldset>
+  <legend>Pipelining</legend>
+  <p class="col-lg-offset-2">Set the stage that will get automatically deployed after this one.</p>
```

Comment:
Ah, I was cutting/pasting from other plugins here. It seems lined up on my screen with the other plugin settings.
Pic or it didn't happen! :)
@sandlerr